### DISCUSSION TITLE
Pagination For the list of output

### DISCUSSION THREAD

**Discussion by dheeraj009joshi** [aiogram/aiogram#1011] [Q&A] [answered]
### Problem

![image](https://user-images.githubusercontent.com/73873399/192305031-8371d8c7-4eae-4996-8549-af753649761b.png)

### Possible solution

Sir i am trying to do pagination of the list of following messages  (eg.  i  need to send three response at a time and put a button after that   on click of that button the next three response should be send 
![image](https://user-images.githubusercontent.com/73873399/192305918-819b05d3-d649-4ae9-8085-3ae91c95fe9e.png)
something like this  

### Additional information

sir it's an urgent request plz let me know how i can achieve that

**Comment by JrooTJunior** [comment, marked as answer]
just read the articles about pagination.

the end-solution depends on youre project dependencies (which DB, connectors, datasources, etc. is used), so, if you have any troubles with this task you can provide more information about your code, show examples, show what implementation options you tried

**Comment by dheeraj009joshi** [comment]
Sir i have tried and i am not using any db I am fetching the results from
scraping and from google

**Reply by mr-st0rm** [reply]
you can try to save the data in the fsm in the form of a sheet and already do pagination as it is convenient for you
